android/external/MakefileTemplate

```diff
@@ -163,20 +163,22 @@ uuid/jni/Android.mk: uuid.src.stamp
 jitter-build: $(LOCAL)/lib/libjitterentropy.a
 
 $(LOCAL)/lib/libjitterentropy.a:
-	$(MAKE) -C $(JITTER_SRC) jitterentropy-static
-	$(MAKE) -C $(JITTER_SRC) jitterentropy-static install-includes PREFIX=$(LOCAL)
-	$(MAKE) -C $(JITTER_SRC) jitterentropy-static install-static PREFIX=$(LOCAL)
+	rsync -av $(JITTER_SRC)/ jitterentropy/
+	$(MAKE) -C jitterentropy AR="$(AR)" CC="$(CC)" CXX="$(CXX)" jitterentropy-static
+	$(MAKE) -C jitterentropy AR="$(AR)" CC="$(CC)" CXX="$(CXX)" jitterentropy-static install-includes PREFIX=$(LOCAL)
+	$(MAKE) -C jitterentropy AR="$(AR)" CC="$(CC)" CXX="$(CXX)" jitterentropy-static install-static PREFIX=$(LOCAL)
```

Comment:
jitterentropy-static can go away, it has already been built

Reply:
Removed.